thefuck/rules/docker_image_being_used_by_container.py

```diff
@@ -0,0 +1,13 @@
+from thefuck.utils import for_app
+
+
+@for_app('docker')
+def match(command):
+    return ('docker' in command.script
+            and 'image is being used by running container' in command.output)
+
+
+def get_new_command(command):
+    container_id = command.output.strip().split(' ')
+    print(container_id[-1])
```

Comment:
Better not to print here, it might be evaluated in some shells =)

Reply:
Oops! Thank you for catching that. I'll remove it!